pyproject.toml

```diff
@@ -88,7 +88,7 @@ format = [
 
 [tool.black]
 line-length = 100
-target-version = ["py37"]
+target-version = ["py38"]
```

Comment:
can we switch to using py311 please?

Reply:
I updated to pypy3.10 because 3.11 is not supported

Reply:
Done !